app/views/shared/navbar/_search_bar.html.erb

```diff
@@ -0,0 +1,8 @@
+
+<label class="input input-bordered input-sm flex items-center gap-2">
+  <input type="text" class="grow" placeholder="Search" autocomplete="off" data-controller="event" data-action="click->event#dispatchEvent" data-event-name-value="spotlight">
```

Comment:
I noticed that if I open the modal and press ESC, this input still has focus, and I can't search from there. Maybe it could be a button styled to look like an input instead of being a real input.

Reply:
I will replace this by just a simple icon. It will look better in mobile too